tests/test_convert.py

```diff
@@ -100,15 +53,14 @@ def test_conversions(tmp_path: Path, package_name, branch):
     )
     srpm_path = next(sg_path.glob("*.src.rpm"))
     assert srpm_path.exists()
-    # TODO: implement `packit prep` and run it here
```

Comment:
thanks!

Reply:
Sorry for the removal. Was that important?

Reply:
I mean, thanks for fixing the TODO and making it configurable :)